pom.xml

```diff
@@ -66,5 +66,20 @@
                 </configuration>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
ok, now you can remove the compiler plugin that you added **above**  (we have it here now)

Reply:
Done